src/Payments.Mvc/ClientApp/containers/CreateInvoiceContainer.tsx

```diff
@@ -297,6 +297,8 @@ export default class CreateInvoiceContainer extends React.Component<IProps, ISta
                 body = JSON.stringify({
                     ccEmails,
                 });
+            } else {
+                body = JSON.stringify({ ccEmail: "" });
```

Comment:
@jSylvestre plural i think?  It's supposed to match up with `ccEmails` above.  On option could be to do `body = JSON.stringify({ ccEmails: i === 0 ? ccEmails : '' });`